http/src/main/java/com/redhat/lightblue/client/http/LightblueHttpClient.java

```diff
@@ -286,5 +296,37 @@ private static HttpTransport defaultHttpClientFromConfig(LightblueClientConfigur
             throw new RuntimeException(e);
         }
     }
+    
+    private Execution getExecutionInstance() {
+        if (execution == null) {
+            synchronized (this) {
+                if (execution == null) {
```

Comment:
Empty execution should be the same as no execution, so yes, whatever is configured in datasources.json.